src/Razor/src/Microsoft.VisualStudio.Editor.Razor/BackgroundDocumentGenerator.cs

```diff
@@ -327,7 +327,6 @@ private void ProjectManager_Changed(object sender, ProjectChangeEventArgs e)
                     }
 
                 case ProjectChangeKind.DocumentAdded:
-                case ProjectChangeKind.DocumentChanged:
```

Comment:
@rynowak do you recall why we thought this was a good idea? lol 😄

Reply:
Because the project snapshots include the content of the documents as well. I'm not sure if any other assumptions change to invalidate this since I wrote it like 3939393 years ago.

I have full faith in the team 🤣

Reply:
Test answered the question. We capture document changed here for the verrrry first state swap from closed -> open when this piece "suppresses" the background document. So can modify this PR to just not reparse related files 😄